lib/rules/no-multi-spaces.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * @fileoverview This rule warns about the usage of extra whitespaces between attributes
+ * @author Armano
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'This rule warns about the usage of extra whitespaces between attributes',
+      category: 'Stylistic Issues',
+      recommended: false
+    },
+    fixable: 'whitespace',  // or "code" or "whitespace"
+    schema: []
+  },
+
+  /**
+   * @param {RuleContext} context - The rule context.
+   * @returns {Object} AST event handlers.
+   */
+  create (context) {
+    function formatValue (token) {
+      switch (token.type) {
+        case 'HTMLSelfClosingTagClose': return '/>'
+        case 'HTMLTagClose': return '>'
+      }
+
+      return token.value
```

Comment:
Ah, also the `value` property of `HTMLAssociation` tokens is empty.
I think `context.getSourceCode().getText(token)` is good for this purpose.